examples/terraform-submodules/gke/variables.tf

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+variable "project" {
+  default = ""
+}
+
+variable "name" {
+  default = "agones-terraform-example"
+}
+
+// Install latest version of agones
+variable "agones_version" {
+  default = ""
+}
+
+variable "machine_type" {
+  default = "n1-standard-4"
+}
+
+// Note: This is the number of gameserver nodes. The Agones module will automatically create an additional
+// two node pools with 1 node each for "agones-system" and "agones-metrics".
+variable "node_count" {
+  default     = "4"
+  description = "The number of gameserver nodes."
+}
+
+variable "zone" {
+  default     = "us-west1-c"
+  description = "The GCP zone to create the cluster in"
+}
+
+variable "network" {
+  description = "The name of the VPC network to attach the cluster and firewall rule to"
```

Comment:
The deleted config has `default     = "default"` for this variable, but that is no longer present, which I think makes this a required variable.

Reply:
That's right, it should be returned